### PR TITLE
another timer fix

### PR DESCRIPTION
**Pull request by ebuchman**

### CODE REVIEW

File: common/repeat_timer.go [hunk 1]
```diff
@@ -31,20 +33,29 @@ func (t *DefaultTicker) Chan() <-chan time.Time {
 	return t.t.C
 }
 
-// Implements Ticker
-func (t *DefaultTicker) Stop() {
+// Implements Ticker. Returns true if the ticker is stopped,
+// false if it was already stopped.
+func (t *DefaultTicker) Stop() bool {
+	if t.stopped {
```
Comment (jaekwon) [inline common/repeat_timer.go]: race condition

**Comment by jaekwon** [comment]
See https://github.com/tendermint/tmlibs/pull/111/files

File: common/repeat_timer.go (hunk 1 above)
Comment (ebuchman) [reply common/repeat_timer.go]: not meant to be called concurrently. 

Locking handled by the RepeatTimer

File: common/repeat_timer.go [hunk 2]
```diff
@@ -8,21 +8,23 @@ import (
 // Ticker is a basic ticker interface.
 type Ticker interface {
 	Chan() <-chan time.Time
-	Stop()
+	Stop() bool
```
Comment (jaekwon) [inline common/repeat_timer.go]: See https://github.com/tendermint/tmlibs/pull/111/files#diff-3edd8f0210da5b9853bae76c67ece2d4R203 .

FYI In tmlibs/common.Service, we return an error which is more forgiving, but seems maybe OK.  To return a bool seems to be more dangerous, as it encourages usage as if it's not an error.

**Pull request closed without merging**